src/test/mir-opt/lower_128bit_test.rs

```diff
@@ -8,9 +8,6 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// asmjs can't even pass i128 as arguments or return values, so ignore it.
-// this will hopefully be fixed by the LLVM 5 upgrade (#43370)
-// ignore-asmjs
```

Comment:
The issue is closed, we'll se if this works, but if it doesn't, it might need a new issue.

Reply:
Since asmjs isn't upgraded here this is probably still an issue but I'm fine waiting for bors too!